consumer/pdatautil/pdatautil.go

```diff
@@ -146,3 +146,21 @@ func TimeseriesAndPointCount(md consumerdata.MetricsData) (int, int) {
 	}
 	return numTimeSeries, numPoints
 }
+
+// Initializes a metric with a metric slice and returns it.
+func InitializeMetricSlice(metricData data.MetricData) pdata.MetricSlice {
+	rms := metricData.ResourceMetrics()
```

Comment:
One thing I'm not sure about it is if I should be including Resource info in the metrics generated by the Host Metrics receiver. For now I've left it empty, but if anyone has any recommendations here or documentation to point me at, that would be appreciated.

Reply:
For the moment don't add this to the public package. Please create an issue to add this later. Same for AddMetric.

Reply:
Issues Created:
- https://github.com/open-telemetry/opentelemetry-collector/issues/863
- https://github.com/open-telemetry/opentelemetry-collector/issues/864